server/include/ins_service.hpp

```diff
@@ -13,6 +13,11 @@
 #include "data_store.hpp"
 #include "localization.hpp"
 #include "types.hpp"
+extern "C"
+{
+#include <WifiNode.h>
+#include <WifiAccessPointLocalConfig.h>
+}
```

Comment:
It is going to be a nightmare to unit tests this. :cry:

Reply:
:) why?